src/near/types/actions.rs

```diff
@@ -0,0 +1,187 @@
+use crate::near::types::PublicKey;
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::AccountId;
+
+use super::{U128, U64};
+
+#[derive(Serialize, Deserialize, Debug, Clone, BorshSerialize, BorshDeserialize, PartialEq, Eq)]
+#[serde(crate = "near_sdk::serde")]
+pub enum Action {
```

Comment:
is there no library from nearcore to reuse here? or it's too many dependencies there?